.gitignore

```diff
@@ -1,13 +1,13 @@
+# dotfiles
+.*
```

Comment:
Smart ! Then I guess you could have removed `.cache` from the list... ;-) That being said, it would be better if this was in a separate commit as it is not related to the `Modifier` improvement here. Same PR is ok but not same commit. I suggest `[IMP] Core: improve gitignore` or something along these lines.